scispacy/hyponym_detector.py

```diff
@@ -22,11 +22,15 @@ class HyponymDetector:
 
     The pipe can be used with an instantiated spacy model like so:
     ```
-    hyponym_pipe = HyponymDetector(extended=True)
-    nlp.add_pipe(hyponym_pipe, last=True)
+    # you first need to add the lemmatizer from the base model
+    nlp_sm = spacy.load('en_core_web_sm')
+    nlp.add_pipe('attribute_ruler', source=nlp_sm, after='tagger')
```

Comment:
Is the `attribute_ruler` actually required here, because we create our own `Matcher` instance below? If it is required, we should probably use the component that it creates, rather than create a new one here. (to be clear I think the Matcher inside the init here is fine)

Reply:
so, the `Matcher` requires a `lemmatizer`. In turn, the `lemmatizer` requires a `morphologizer`. the alternative to a `morphologizer` is a `tagger` followed by an `attribute_ruler`. the `attribute_ruler` has the rules for setting `pos` and `morph`, whereas the tagger just sets `tag`.

Reply:
So if we just include the lemmatizer in the base model, this would all go away?

Reply:
yeah, would include the lemmatizer and the attribute_ruler. this would give us the rule based lemmatization again

Reply:
gotcha, that makes me even more sure that figuring that out is worthwhile 👍

Reply:
It is either as easy as tweaking the config and rerunning the whole pipeline, or there is rake waiting for me to step on it, we shall see